### ISSUE TITLE
subdivisions include in export

### ISSUE THREAD

**Issue by msfielder** [soupday/cc_blender_tools#221]
is there any way to use a  cc4 subdivided mesh and export this including those subdivisions? ... for roundtrip use? ... 
tried to multires mesh in blender, then sculpt it, then export it again to cc ... file gets refused ... as mesh-data has changed i guess ..
so i thought that it would make sense to use the subdiv from cc ... but can not get these to export ...

**Comment by soupday** [comment]
You can export with subdivision, scroll to the bottom of the export window, there's an option at the end. But you can't bring it back in with the round trip.

**Comment by msfielder** [comment]
yes, i m trying to get the subdiv to export through the blender-pipepline, so i can do roundtrip. so that does not work, right?

**Comment by msfielder** [comment]
got a project i would need this for. it s got a lowlow budget, but i d gladly pay some money for this functionality.
